src/features/duplicates/hooks/useDuplicatesMutations.tsx

```diff
@@ -0,0 +1,29 @@
+import { dismissedDuplicate, ZetkinDuplicate } from '../store';
+import { useApiClient, useAppDispatch } from 'core/hooks';
+
+type DuplicatesMutationsReturn = {
+  dismissDuplicate: (duplicateId: number) => void;
+};
+
+export default function useDuplicatesMutations(
+  orgId: number
+): DuplicatesMutationsReturn {
+  const apiClient = useApiClient();
+  const dispatch = useAppDispatch();
+
+  const dismissDuplicate = async (duplicateId: number) => {
+    const dismissedDate = new Date().toISOString();
+    await apiClient
+      .patch<ZetkinDuplicate>(
+        `/api/orgs/${orgId}/people/duplicates/${duplicateId}`,
+        { dismissed: dismissedDate }
```

Comment:
I checked this API call and it doesn't work now because it's the backend who adds the timestamp of the date. So we only need to send {dismissed: true } when we want to dismiss a duplicate.